### ISSUE TITLE
Storage Layouts - custom partitions with interactive selection for easy of update.

### ISSUE THREAD

**Issue by Po-Pratik** [canonical/autoinstall-desktop#10]
We are looking to get below preseed file converted into autoinstall.

**PRESEED Used for Ubuntu 18.04 LTS**
d-i partman-auto/method string lvm

d-i partman-lvm/device_remove_lvm boolean true
d-i partman-lvm/confirm boolean true
d-i partman-lvm/confirm_nooverwrite boolean true
d-i partman-lvm/device_remove_lvm boolean true
d-i partman-auto-lvm/guided_size string max
d-i partman-basicfilesystems/no_swap boolean false

# For LVM partitioning, you can select how much of the volume group to use
# for logical volumes.
d-i partman-auto/choose_recipe select boot-root

d-i partman-auto/expert_recipe string            	\
    boot-root ::                                 	\
		512	2000	2048 ext4                      	\
			$primary{ } $bootable{ }             	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /boot }                  	\
        .		                                 	\
        2048 128 8192 linux-swap                 	\
		    $lvmok{ }                            	\
            method{ swap } format{ }             	\
        .                                        	\
        5120	10000	-1 ext4                    	\
		    $lvmok{ }								\
            method{ format } format{ }          	\
            use_filesystem{ } filesystem{ ext4 }	\
            mountpoint{ / }                     	\
        .                                       	\
        5120	30000	25600 ext4                 	\
		    $lvmok{ }								\
            method{ format } format{ }          	\
            use_filesystem{ } filesystem{ ext4 }	\
            mountpoint{ /home }                    	\
        .                                       	\
        1024	5000	4096 ext4                  	\
		    $lvmok{ }							 	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /usr/local/safesquid }   	\
        .                                        	\
        5120	10000	8192 ext4                 	\
		    $lvmok{ }							 	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /var/lib/safesquid }		\
        .                                        	\
        512	5000	4096 ext4                      	\
		    $lvmok{ }							 	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /opt/safesquid }			\
        .                                        	\
        1024	5000	-1 ext4                     \
		    $lvmok{ }                            	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /var/cache/safesquid }   	\
        .										 	\
        1024	45000	-1 ext4                     \
		    $lvmok{ }                            	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /var/log/safesquid }     	\
        .										 	\
        512	16000	15360 ext4                     \
		    $lvmok{ }                            	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /var/www/safesquid }     	\
        .										 	\
		2048	35000	-1  ext4                   	\
		    $lvmok{ }                            	\
            method{ format } format{ }           	\
            use_filesystem{ } filesystem{ ext4 } 	\
            mountpoint{ /var/db/safesquid }     	\
        .
and does autoinstall provides an option wherein the users can update the storage size for each partition using interactive menu.

**Comment by kirkog86** [comment]
No. storage: section is preconfigured in user-data file and can't by interactive
